docs/user/devices.md

```diff
@@ -156,30 +156,46 @@ backup of the Node-RED flows and configuration.
 This model allows you to develop your flows in FlowForge and only push it out
 to the registered devices when you're happy with what you've created.
 
-## Editing the Node-RED flows directly on a device
+## Editing the Node-RED flows on a device
```

Comment:
docs update title "Editing the Node-RED flows directly on a device" -> "Editing the Node-RED flows on a device" will break link in `packages/flowforge/docs/user/snapshots.md`
need to change link to `See [Working with Devices](./devices.md#editing-the-node-red-flows-on-a-device) for more information.`

Reply:
I'm guessing `Test Documentation Links / test (pull_request)` doesnt check internal MD links?